### ISSUE TITLE
Cider Commands broken after update

### ISSUE THREAD

**Issue by nathan-rogers** [aplteam/Cider#74]
After a fresh `]activate all -reset`, restarting version 19, I run the following user commands: 

```
      ]TATIN.updateTatin

Your version of Tatin is 0.104.1
Latest published version is 0.112.0
Would you like to download and install 0.112.0 ? (Y/n) Y
Tatin successfully updated to version 0.112.0 in C:\Users\natha\Documents\MyUCMDs\Tatin

      ]CidER.updateCider

Are you sure that you want to update Cider to version 0.42.1 ? (Y/n) Y
Cider was successfully updated to version 0.42.1 in C:\Users\natha\Documents\Dyalog APL-64 19.0 Unicode Files/SessionExten
      sions/CiderTatin/Cider

      ]Cider.createProject c:\d\myproject
* Command Execution Failed: VALUE ERROR: Undefined name: UC
```

I can `]CiderCreateProject` before I run the updates, but I'm updating because there is a bug in an older version when installing nuget packages. After the update, the user command script seems to be failing here:

```
      ]Cider.createProject c:\d\game
VALUE ERROR: Undefined name: UC
List[4] :If 3=ref.UC.⎕NC'List'
                  ∧
```
![image](https://github.com/user-attachments/assets/7c368bd1-ec76-4c30-8216-42e75443515c)

**Comment by aplteam** [comment]
Please try this in a freshly started copy of Dyalog.

**Comment by nathan-rogers** [comment]
Yes, I've freshly installed Dyalog, and this was done in a brand new session of Dyalog.

**Comment by aplteam** [comment]
When I first looked at this I overlooked the cause of the problem, sorry: with 19.0 Cider and Tatin are no longer installed into the MyUCMDs\ folder but must be installed into C:\Users\<username>\Documents\Dyalog APL Files\SessionExtensions

Please execute this statement:

```
⍪';'(≠⊆⊢)⎕se.SALT.Settings 'cmddir'
```

If the MyUCMDs\ folder is on the list and comes before the SessionExtensions\ folder then that explains what you are experiencing: the user command is detected in MyUCMDs\ and SALT (in charge for finding and establishing user commands) won't look any further.

If Cider/Tatin in MyUCMDs\ is a leftover from when you used Cider/Tatin with 18.0 or 18.2, then simply deleting them from the MyUCMDs\ folder will solve the problem.

If you want to use Cider/Tatin with 18.0 or 18.2 as well then please read "6. Updating Tatin" in the InstallingAndUpdatingTheTatinClient document: <https://tatin.dev/Assets//docs/InstallingAndUpdatingTheTatinClient.html#updating-tatin>

It explains how to make Cider/Tatin available in 18.0 or 18.2 after deleting both from the MyUCMDs\ folder.

I am going to close this issue. If you need further assistance, please communicate with me via kai@aplteam.com
